### PR TITLE
MM-1554 Removes the delete icon for the first entry in the 'Invite New Members' form

### PR DESCRIPTION
**Pull request by rgarmsen2295**
Can no longer clear or remove the first entry in the 'Invite New Members' form.  Also did some minor cosmetic refactoring that includes switching variable names to be camel case, and other minor cases such as double to single quotes.

### CODE REVIEW

**Comment by hmhealey** [comment]
Looks good to me

File: web/react/components/invite_member_modal.jsx [hunk 1]
```diff
@@ -89,146 +90,149 @@ module.exports = React.createClass({
...
+            var inviteIds = this.state.inviteIds;
+            for (var i = 0; i < inviteIds.length; i++) {
+                var index = inviteIds[i];
+                var emailError = this.state.emailErrors[index] ? <label className='control-label'>{this.state.emailErrors[index]}</label> : null;
+                var firstNameError = this.state.firstNameErrors[index] ? <label className='control-label'>{this.state.firstNameErrors[index]}</label> : null;
+                var lastNameError = this.state.lastNameErrors[index] ? <label className='control-label'>{this.state.lastNameErrors[index]}</label> : null;
+
+                inviteSections[index] = (
+                    <div key={'key' + index}>
+                    {index ?
```
Comment (crspeller) [inline web/react/components/invite_member_modal.jsx]: These ternaries will need to be removed to conform to the style guide. Probably should do that now.

Comment (rgarmsen2295) [inline web/react/components/invite_member_modal.jsx]: Ah yeah fair enough, will change that.

**Comment by rgarmsen2295** [comment]
Completed additional refactoring, please take another look when you have a chance @crspeller

**Comment by rgarmsen2295** [comment]
Ok ready now @crspeller

**Comment by crspeller** [comment]
Looks good.

**Pull request merged**